bin/generate-gutenboarding-design-thumbnails.js

```diff
@@ -59,10 +66,14 @@ async function run() {
 		return;
 	}
 	console.log( `Processing ${ designs.featured.length } designs...` );
+	console.log( `for each of ${ localeSlugs.join( ', ' ) } (${ localeSlugs.length } locales)...` );
 
+	// ( localeSlugs || [ '' ] ).forEach( async ( locale ) =>
 	await Promise.all(
```

Comment:
Is `Promise.all()`  problematic for this many tasks in parallel, or should we have a simple queue mechanism? ([example](https://caolan.github.io/async/v3/docs.html#queue))